src/drc/api/serializers.py

```diff
@@ -397,6 +441,58 @@ def save(self, **kwargs):
         return self.instance
 
 
+class CompleteEnkelvoudigInformatieObjectSerializer(serializers.ModelSerializer):
+    """
+    Serializer for complete action of EnkelvoudigInformatieObject model
+    """
+    url = serializers.HyperlinkedIdentityField(
+        view_name='enkelvoudiginformatieobject-detail',
+        lookup_field='uuid'
+    )
+    inhoud = ViewFileFile(view_name='enkelvoudiginformatieobject-download', read_only=True)
+
```

Comment:
I'm missing the lock ID to provide that proves _you_ are the one marking the upload as completed.

Reply:
`lock` is added to the `BestandsDeelSerializer`